390 tautology and internals

ShiriMoran opened the pull request.

adisos commented:
is there a relevant issue open as followup for this PR?

adisos commented on pkg/synthesis/tests_expected_output/abstract_models/ExampleInternalWithInterDenyAllowWithSegments.txt:
```diff
@@ -11,18 +11,20 @@ real-group    |           |vm-no-address1, vm-no-address2
+Original allow rule priority |Rule id |Src                                                                      |Dst                    |Connection
+0                            |1005    |(segment seg_2-3 and not in segment seg_1)                               |(IP addr in 0.0.0.0/0) |All Connections
+1                            |1007    |(segment seg-6-7 and not in segment seg_4-5)                             |(IP addr in 0.0.0.0/0) |All Connections
+2                            |2000    |(segment seg-8)                                                          |(IP addr is external)  |All Connections
```
how can dest be "IP addr is external" for an inbound rule?

ShiriMoran replied:
right. fixed

ShiriMoran commented:
> is there a relevant issue open as followup for this PR?

you mean for the network policies?

adisos commented:
> > is there a relevant issue open as followup for this PR?
> 
> you mean for the network policies?

yes

adisos commented:
@haim-kermany , please review changes in `synthesis/K8sPolicies.go` and in `synthesis/createNSXPolicies.go`

haim-kermany commented:
"ANY" is now not checked. 
there there some tests that does not have "ANY"?

ShiriMoran commented:
> > is there a relevant issue open as followup for this PR?
> 
> you mean for the network policies?

#401

ShiriMoran commented:
> "ANY" is now not checked. there there some tests that does not have "ANY"?



> "ANY" is now not checked. there there some tests that does not have "ANY"?



> "ANY" is now not checked. there there some tests that does not have "ANY"?
Right. I added by mistake "IsAllGroup()" to "policies not supported yet". Fixed.

haim-kermany commented:
I aproved my files
